index.html

```diff
@@ -80,11 +82,27 @@ <h1>Visualization explanation</h1>
 
       <h1>Data Analysis</h1>
       <p><em>Summary of data, data types, and data preprocessing.</em></p>
+      <p>According to the data, the time it takes to cross from one side of the park to the other
+        without jaywalking is almost 11 times greater than crossing the street directly. (based on avg_travel_times.csv)
+        However, the average time it takes to cross a crosswalk is quite similar to that of jaywalking, but is
+        much safer. Even if people cross when there is no signal to walk, it is still safer than no crosswalk.
+      </p>
+
+
 
-      <p><em>Expectation: Data sources match the problem statement and are appropriate. These descriptions should be very explicit so someone could read your page and properly reproduce your results.</em></p>
 
       <h1>Task Analysis</h1>
 
+      <p>We wish to have an interactive map along with some graphs that interact with eachother.
+        The map will be used to demonstrate distance and give a spacial representation of the walking distance
+        between the parks and walking distance to get between them.
+      </p>
+      <p>
+        We also wish to show graphs that correspond to map and show how selecting something from the
+        graph affects the other graphs, as well as the map itself. This can be used to analyze the
```

Comment:
we should probably actually put our task table here with a paragraph that sums it up. I don't think what you have here is more of a design process description